src/llvm-coverage-instrumentation.md

```diff
@@ -82,7 +83,7 @@ a span of code ([`CodeRegion`][code-region]). It counts the number of times a
 branch is executed, and also specifies the exact location of that code span in
 the Rust source code.
 
-Note that many of these `Coverage` statements will *not* be converted into
+Note that many of these `Coverage` statements will _not_ be converted into
```

Comment:
Why change this?

Reply:
Sorry. I didn't change this intentionally. My editor seems to have aggressive re-formatting options turned on, and I assumed these were from some Rust-specific settings (like how the editor picks up rust formatting from the committed `rustfmt.toml`).

I'll revert any unnecessary changes.

Reply:
No worries, thanks for taking care of it :)

Reply:
Just FYI, I reverted the unintended changes made by the formatter, but I also compared the style in this doc to the style in the rest of the Rustc Dev Guide.

This doc was not internally consistent (italics/emphasis used both `_word_` and `*word*` with the same result, and bulleted items used both `*` and `-`. I chose the most common usage in the dev guide and make this document consistent (`_` and `-`, respectively).

Reply:
Oh, and I rewrapped lines that were too long, compared to a standard line length used in most of the document.